cli/args/flags.rs

```diff
@@ -1386,6 +1387,12 @@ Ignore linting a file by adding an ignore comment at the top of the file:
         .help("Output lint result in JSON format")
         .takes_value(false),
     )
+    .arg(
+      Arg::new("compact")
+        .long("compact")
+        .help("Output lint result in compact format")
+        .takes_value(false),
```

Comment:
This arg should have `conflicts_with()` setting, so users can't use `--compat` with `--json`

Reply:
good catch. Its been updated